tests/resetDB.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const Knex = require("knex");
+
+const AppConstants = require("../app-constants");
+const DB = require("../db/DB");
+const knexConfig = require("../db/knexfile");
+const {seed} = require("../db/seeds/test_subscribers");
+
+
+// (Re-)create DB connection at the beginning of each test suite
+beforeAll(async () => {
+  DB.createConnection();
+});
+
+
+beforeEach(async () => {
```

Comment:
Nit: add a code comment above saving we’re nuking your `subscribers` table before each test?